Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -501,14 +540,34 @@ def get_urlscan_submit_results_polling(client, uuid):
 
 
 def urlscan_submit_command(client):
+    execution_metrics = ExecutionMetrics()
+    command_results = []
+    items_to_schedule = []
+    calling_context = demisto.callingContext.get('context', {})
+    sm = get_schedule_metadata(context=calling_context)
```

Comment:
move the logic to common server. Wrap it into something like `is_polling_first_iteration()` or some other similar name 
which will replace `not sm.get('is_polling')`